### PR TITLE
Create CSS-Position-Properties.md

### PR DESCRIPTION
**Pull request by Jaya3112**
<hr>

## Description 📜

CSS Position Properties #6561

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev #HTML/CSS/JS

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

### CODE REVIEW

**Review by SauravMukherjee44 — APPROVED** [review]

**Review by praveenscience — CHANGES_REQUESTED** [review]
Kindly make the following requested changes.

@SauravMukherjee44 How did you miss these? We're doing a Markdown not a text document.

/cc @2024-SANDHYA @salil-naik @kashishahuja2002

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Can you also add some SS for better understanding?

File: Web_Development/FrontEnd/CSS-Position-Properties.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+## 1. Static Position:
+
+- It is by default position property of a HTML element.
+- It doesnot get affected by the top, bottom, right and left properties.
+- With static position the element is positioned according to normal flow of document.
+
+**Example**
+
+```
```
Comment (praveenscience) [inline Web_Development/FrontEnd/CSS-Position-Properties.md:13]: Should be for all:

    ```css

File: Web_Development/FrontEnd/CSS-Position-Properties.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+.static {
+  position: static;
+  border: 2px solid black;
+}
+```
+
+  ## 2.  Relative Position:
```
Comment (praveenscience) [inline Web_Development/FrontEnd/CSS-Position-Properties.md:20]: The space is messing it up.

**Review by praveenscience — CHANGES_REQUESTED** [review]

**Review by praveenscience — APPROVED** [review]
There are too many grammatical changes, but go for it. 😅

**Pull request merged**